services/production-tools/templates/secrets.yaml

```diff
@@ -0,0 +1,8 @@
+---
+apiVersion: ricoberger.de/v1alpha1
+kind: VaultSecret
+metadata:
+  name: butler-secret
+spec:
+  path: {{ .Values.butler_secret_path }}
```

Comment:
This shouldn't be a separate configuration parameter and instead should use `global.vaultSecretsPath` (see how Gafaelfawr does this).